arm/Microsoft.EventHub/namespaces/eventhubs/.bicep/nested_rbac.bicep

```diff
@@ -25,10 +25,9 @@ resource eventHub 'Microsoft.EventHub/namespaces/eventhubs@2021-06-01-preview' e
 }
 
 resource roleAssignment 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = [for principalId in roleAssignmentObj.principalIds: {
-  name: guid(eventHub.name, principalId, roleAssignmentObj.roleDefinitionIdOrName)
+  name: guid(resourceName, principalId, roleAssignmentObj.roleDefinitionIdOrName)
   properties: {
     roleDefinitionId: contains(builtInRoleNames, roleAssignmentObj.roleDefinitionIdOrName) ? builtInRoleNames[roleAssignmentObj.roleDefinitionIdOrName] : roleAssignmentObj.roleDefinitionIdOrName
     principalId: principalId
   }
-  scope: eventHub
```

Comment:
Why are you removing this? If you remove this the scope will be the deployment scope of RG i think.

Reply:
No idea. Added it again